JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -2580,7 +2586,7 @@ AnovaRepeatedMeasures <- function(dataset=NULL, options, perform="run", callback
   
   simpleEffectsTable[["schema"]] <- list(fields=fields)
   
-  if (perform == "run" && status$ready && status$error == FALSE)  {
+  if (perform == "run" && status$ready && status$error == FALSE && class(fullModel) != "try-error")  {
```

Comment:
I've made a funciton isTryError in common.R, so you might be interested to implement this as: 

!isTryError(fullModel)